src/test/java/com/jcabi/github/mock/MkIssueTest.java

```diff
@@ -253,6 +255,53 @@ public void assignsUser() throws Exception {
     }
 
     /**
+     * MkIssue creates a closed event when closing an issue.
```

Comment:
@cvrebert documentation for a test should start with "MkIssue can create..." in this case.